02_bubble/selva/bubblesort_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutputpwd(t *testing.T) {
```

Comment:
Write a function `TestBubble` that tests `bubble` directly.
Include corner cases: empty slice, 1-element slice, sorted slice, reverse-sorted slice, slice with duplicated elements.
Consider using [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests)